src/App.tsx

```diff
@@ -1,42 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-export const tabs = [
+import { Tabs } from './components/Tabs';
+import { Tab } from './types/Tab';
```

Comment:
you dont need this row in `App`

Reply:
Agree, it's extra, ts already know about this
I'll remove it after the mentor's review